lisa/tools/fdisk.py

```diff
@@ -1,18 +1,11 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
-from enum import Enum
 from typing import cast
 
 from lisa.executable import Tool
 from lisa.operating_system import Posix
-from lisa.util import LisaException
 
-from .mkfs import Mkfsext, Mkfsxfs
-
-FileSystem = Enum(
-    "mkfs",
-    ["xfs", "ext2", "ext3", "ext4"],
-)
+from .mkfs import FileSystem, Mkfs
```

Comment:
The description of this commit "Refine mkfs." is too simple to the change. It needs to specify what's problem fixed, or what functionality added.